### PR TITLE
Test Suite

### PR DESCRIPTION
**Pull request by mhemesath**
Our team is going to start using R2 pretty heavily, so I've started a test suite to ensure everything is covered.  This pull request isn't ready to be merged yet, but thought I'd push it out so you can comment on the direction its headed.

### CODE REVIEW

**Comment by mhemesath** [comment]
Also, in the tests I've pushed so far, a few of the tests are failing around border-radius. We need to add better support for examples like

```
border-radius: 0 1px 2px;
```

**Comment by ded** [comment]
rock on. this is a good start!

File: tests/tests.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    assert.equal(swap('p{border-width:0;}'), 'p{border-width:0;}', 'border-width: 0 => border-width: 0')
+    assert.equal(swap('p{border-width:0 1px;}'), 'p{border-width:0 1px;}', 'border-width: 0 1px => border-width: 0 1px')
+    assert.equal(swap('p{border-width:0 1px 2px;}'), 'p{border-width:0 1px 2px;}', 'border-width: 0 1px 2px => border-width: 0 1px 2px')
+    assert.equal(swap('p{border-width:0 1px 2px 3px;}'), 'p{border-width:0 3px 2px 1px;}', 'border-width: 0 1px 2px 3px => border-width: 0 3px 2px 1px')
+  })
+})
+
+sink('border-radius', function(test, ok, before, after, assert) {
+  
+  test('should swap border-radius', 12, function() {
```
Comment (ded) [inline tests/tests.js]: just as a tip, y ou can use the `done()` callback instead of providing a high number of assertions. For example, see here: https://github.com/ded/sink-test/blob/master/tests/tests.js#L74

Comment (mhemesath) [inline tests/tests.js]: nice! Thanks, this is my first time using sink.... not too shabby.    :)

**Comment by mhemesath** [comment]
Ok, I have tests in place for all existing functionality.  There are other propeties (e.g. background-position) that need to be in here, but we'll log that as an issue soon and add a test with the pull request.

I can squash this into a single commit if you like.. otherwise it should be ready to merge.

**Comment by mhemesath** [comment]
It might be cool to also hook this into Travis CI.

**Comment by ded** [comment]
yeah. a few of my other projects are already on Travis. considering this one is fully server-side (it was a pain to setup some of the DOM related stuff with webkit driver, phantom, etc)... this setup should be fairly easy

**Comment by ded** [comment]
border-radius was failing for me. but that's alright for now. at least there are tests!

**Comment by mhemesath** [comment]
Yeah, there are a few edge cases we need to fix.  Specifically:

```
border-radius: 0 1px 2px
```
